.github/workflows/sphinx_doc.yaml

```diff
@@ -0,0 +1,12 @@
+name: "Pull Request Docs Check"
+on: 
+- pull_request
+
+jobs:
+  docs:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: ammaraskar/sphinx-action@master
```

Comment:
Hmm, code in this action hasn't been changed for 2 years. Wouldn't it be better just run sphinx from command line? I mean replace pdoc invocation below with the relevant sphinx command?
https://github.com/oVirt/python-ovirt-engine-sdk4/blob/main/.github/workflows/create_docs.yaml#L53